import-export-cli/integration/testutils/operationPolicy_testUtils.go

```diff
@@ -0,0 +1,552 @@
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package testutils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/apim"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/base"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	PolicyVersionKey = "version"
+	PolicyNameKey    = "name"
+	PolicyIDReset    = "-1"
+	PolicyIdKey      = "id"
+)
+
+const (
+	DefaultPolicyListSize       = "-1"
+	DefaultPolicyListOffsetSize = "-1"
+	DefaultPolicyListLimit      = "25"
+	DefaultPolicyListOffset     = "0"
+)
+
+// ValidateAPIPolicyExportImport : Validates Exporting API Policy from source env and Importing to destination env
+func ValidateAPIPolicyExportImport(t *testing.T, args *PolicyImportExportTestArgs) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[PolicyNameKey])
+	policyVersion := fmt.Sprintf("%v", args.Policy[PolicyVersionKey])
+
+	args.SrcAPIM.Login(args.CtlUser.Username, args.CtlUser.Password)
+
+	exportedOutput, _ := exportAPIPolicy(t, policyName, args.SrcAPIM.GetEnvName())
+
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+
+	// fmt.Println("Exported Path: ", args.ImportFilePath)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+
+	args.SrcAPIM.DeleteAPIPolicy(fmt.Sprintf("%v", args.Policy[PolicyIdKey]), "Export")
+
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Import API Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+
+	importedOutput, err := importAPIPolicy(t, args)
+
+	assert.Nil(t, err, "Error while importing the API Policy")
+	assert.Contains(t, importedOutput, "Successfully Imported API Policy.")
+	// Give time for newly imported API Policy to get indexed
+	base.WaitForIndexing()
+
+	args.DestAPIM.Login(args.CtlUser.Username, args.CtlUser.Password)
+
+	policyId := args.DestAPIM.GetAPIPolicyID(t, policyName, policyVersion)
+
+	// Get API Policy from env 2
+	importedPolicy := args.DestAPIM.GetAPIPolicy(policyId)
+
+	// Validate env 1 and env 2 policy is equal
+	ValidatePoliciesEqual(t, args, importedPolicy)
+	cleanUpImportExportPolicies(t, args, policyId)
```

Comment:
We shouldn't use explicit functions like this. Need to add t.Cleanups inside exportAPIPolicy() and importAPIPolicy() functions.